cluster/libvirt-coreos/util.sh

```diff
@@ -220,6 +220,13 @@ function kube-up {
     rm $domain_xml
   done
 
+  export KUBECONFIG="$ROOT/.kubeconfig"
```

Comment:
In cluster/gce/util.sh this is set to ${HOME}/.kube/.kubeconfig. Why did you opt to put it under cluster/libvert-coreos here?